MartinP460/react-denmark-map
# Optimized and standarized path definitions

### Proposed changes

Applied optimization techniques to the underlying SVGs rendering the map and made them all the same size (meaning that they have the same fixed viewbox dimensions of 1000 x 1215). The optimization resulted in an average reduction in the size of the SVGs by ~30%. This will reduce the bundle size and it also means that all renditions of the map can load quicker when used in production.

### How to test

1. Open Storybook and verify that all the maps are rendered correctly.
2. Verify that the tests don't fail.
